addons/pinniped/config-controller/controllers/controllers.go

```diff
@@ -86,35 +87,36 @@ func (c *PinnipedController) Reconcile(ctx context.Context, req ctrl.Request) (r
 
 	if err := c.client.Get(ctx, client.ObjectKeyFromObject(pinnipedInfoCM), pinnipedInfoCM); err != nil {
 		if !k8serror.IsNotFound(err) {
-			log.Error(err, "Error getting pinniped-info configmap")
+			log.Error(err, "error getting pinniped-info configmap")
 			return reconcile.Result{}, err
 		}
 
-		log.Info("pinniped-info configmap not found, setting value to nil")
+		log.V(1).Info("pinniped-info configmap not found, setting value to nil")
```

Comment:
I'm a little inclined to do like `log.V(4)` or something higher, put this in our deployment.yaml for now, but when we are really to merge back into TF for release we can drop it from the deployment.yaml

Reply:
ie, for now during dev we keep the log level high.

Reply:
yeah, per the controller-runtime best practice (https://github.com/kubernetes-sigs/controller-runtime/blob/9ee63fc65a9720568f74df9901b16883621636b4/TMP-LOGGING.md), I kept everything at `V(1)` or no V (`V(0)`)